RoaringBitmap/src/main/java/org/roaringbitmap/BitSetUtil.java

```diff
@@ -15,7 +16,7 @@ public class BitSetUtil {
 
   // a block consists has a maximum of 1024 words, each representing 64 bits,
   // thus representing at maximum 65536 bits
-  static final private int BLOCK_LENGTH = BitmapContainer.MAX_CAPACITY / Long.SIZE; //
+  public static final int BLOCK_LENGTH = BitmapContainer.MAX_CAPACITY / Long.SIZE; //
```

Comment:
Is it necessary to make BLOCK_LENGTH public?

Reply:
Didn't see any other neat way to expose the information. 
Since the user can provide the buffer, they need to know atleast what size it needs to be. 
Can just mention it in Javadoc, the bounds check will anyways raise error if bad sized buffer is provided.